packages/core/chrome/core-chrome-browser-internal/src/ui/project/navigation.tsx

```diff
@@ -7,54 +7,16 @@
  */
 
 import React from 'react';
-import { css } from '@emotion/react';
-import { EuiCollapsibleNav, EuiCollapsibleNavProps } from '@elastic/eui';
-
-const SIZE_EXPANDED = 248;
-const SIZE_COLLAPSED = 0;
-
-export interface ProjectNavigationProps {
-  isOpen: boolean;
-  closeNav: () => void;
-  button: EuiCollapsibleNavProps['button'];
-}
-
-export const ProjectNavigation: React.FC<ProjectNavigationProps> = ({
-  children,
-  isOpen,
-  closeNav,
-  button,
-}) => {
-  const collabsibleNavCSS = css`
-    border-inline-end-width: 1,
-    display: flex,
-    flex-direction: row,
-  `;
-
-  const DOCKED_BREAKPOINT = 's' as const;
-  const isVisible = isOpen;
+import { EuiCollapsibleNavBeta } from '@elastic/eui';
 
+export const ProjectNavigation: React.FC = ({ children }) => {
   return (
     <>
       {
-        /* must render the tree to initialize the navigation, even if it shouldn't be visible */
-        !isOpen && <div hidden>{children}</div>
+        /* must render the tree to initialize the navigation, even if hidden internally in EUI */
+        <div hidden>{children}</div>
```

Comment:
I think it would be better to use `display:none` or `display:block` on `EuiCollapsibleNavBeta` and remove those L16 and L17 to avoid re-rendering the nav tree every time.

Reply:
Looking back in the history of this file, this code was added to solve `Issue 3. Navigation is not initialized when Kibana loaded with hidden navigation` in https://github.com/elastic/kibana/pull/163700

Reply:
> I think it would be better to use display:none or display:block on EuiCollapsibleNavBeta and remove those L16 and L17 to avoid re-rendering the nav tree every time.

@sebelga this will work. The old code used `isOpen && children` to stop rendering when the nav is collapsed, which had a bug where the navigation tree would not get initialized when you refresh the page with hidden navigation, causing breadcrumbs and maybe other things to not work. (@Dosant can you confirm I understand that issue correctly?) The new code doesn't have that problem since the navigation is hidden using CSS. 

Added 879b29fd910 to clean that up, and also also unblocks this PR regarding a separate issue.